src/main/kotlin/com/hibob/feedback/dao/FeedbackDao.kt

```diff
@@ -44,7 +44,7 @@ class FeedbackDao(private val sql: DSLContext) {
 
     fun createFeedback(feedback: FeedbackInput, activeUser: ActiveUser): UUID {
         return sql.insertInto(feedbackTables)
-            .set(feedbackTables.employeeId, activeUser.employeeId)
+            .set(feedbackTables.employeeId, if (feedback.isAnonymous) null else activeUser.employeeId)
```

Comment:
does this change affect your tests? it should...

Reply:
Its affecting my tests. i forget to push it